chagrade/settings/base.py

```diff
@@ -140,12 +140,26 @@
     # "apps.chahub_auth.oauth_backends.ChahubOAuth2",
     "django.contrib.auth.backends.ModelBackend",
     "apps.profiles.auth_backends.EmailBackend",
+    "social_core.backends.github.GithubOAuth2",
+
 )
 
 # =========================================================================
 # Social Auth
 # =========================================================================
 
+SOCIAL_AUTH_PIPELINE = (
```

Comment:
Let's pull the pipeline steps we aren't using for now

Reply:
Done.